cli/src/display_bytecode.rs

```diff
@@ -0,0 +1,152 @@
+use compiler::bytecode::Opcode;
+use owo_colors::OwoColorize;
+
+pub(crate) fn display_bytecode(bytes: &[u8]) {
+    println!("{}", "Bytecode: ".green());
+    let (ip, constants) = display_constant_pool(bytes);
+    println!();
+    display_byte_instructions(constants, &bytes[ip..])
+}
+
+fn display_byte_instructions(constants: Vec<String>, bytes: &[u8]) {
+    println!("\t{}", "Instructions".green());
+    let mut ip = 0;
+    let bytes_len = bytes.len();
+
+    let byte_padding = bytes_len.to_string().len();
+
+    while ip < bytes_len {
+        let opcode: Opcode = bytes[ip].try_into().expect("not an opcode");
+        macro_rules! get_usize {
+            () => {
+                usize::from_be_bytes((&bytes[ip..ip + 8]).try_into().unwrap())
+            };
+        }
```

Comment:
```suggestion
        macro_rules! get_fixed_slice {
            ($size:expr) => {
                bytes[ip..ip + $size].try_into().unwrap()
            };
        }
        macro_rules! get_usize {
            () => {
                usize::from_be_bytes(get_fixed_slice!(8))
            };
        }
```
You can use `get_fixed_slice!` to get float values.

Reply:
not worth it as it is used only one time for PushFloat